monkey/monkey_island/config_loader.py

```diff
@@ -1,11 +1,20 @@
 import json
 import os
 
-from monkey_island.cc.server_utils.consts import DEFAULT_DATA_DIR
+import monkey_island.cc.environment.server_config_generator as server_config_generator
+from monkey_island.cc.server_utils.consts import DEFAULT_DATA_DIR, DEFAULT_SERVER_CONFIG_PATH
 
 DEFAULT_LOG_LEVEL = "INFO"
 
 
+def create_default_server_config_path():
```

Comment:
Here's some things that don't make sense:
1. This method creates a file, not a path (bad naming + no type hinting)
2. Why are we creating config in `config_loader.py`? Loader presumes that we only load
3. Creating data dir should be a responsibility of some setuper, config loader should be concerned with config only?
4. Creating a default data dir if one does't exist sounds like a separate function

Reply:
1. Changed.
2. Moved to monkey_island/cc/environment/server_config_generator.py.
3. Moved to monkey_island/cc/environment/data_dir_generator.py.
4. Extracted in monkey_island/cc/environment/data_dir_generator.py.
5. When creating a default data dir, we only want to create one directory since the parent is either $HOME or %AppData% whereas `create_data_dir()` in monkey_island.py will create parent directories too if they don't exist (since it takes a user argument).